rules/prefer-event-target.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const MESSAGE_ID = 'prefer-event-target';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `EventTarget` over `EventEmitter`.',
+};
+
+const eventEmitterClassSelector = [
+	':matches(ClassDeclaration, ClassExpression)',
+	'[superClass.name="EventEmitter"]',
+	'[body.type="ClassBody"]',
```

Comment:
You added this body type check, but not tests covers this, is it possible to have different body here? Maybe in TypeScript? Please test it.

Reply:
> You added this body type check, but not tests covers this, is it possible to have different body here? Maybe in TypeScript? Please test it.

I'm sorry, but I think I actually don't understand this feedback.

Is it possible to handle typescript with this lib?
I tried to insert a simple typescript statement (`let a: number = 3;`) and tested, but it seems like parsing error occurs.

I think class declaration's with other body type is not possible, according to [this ECMA document](https://tc39.es/ecma262/#sec-class-definitions) in Javascript,
In case of Typescript, [this specification's 117 page](https://javascript.xgqfrms.xyz/pdfs/TypeScript%20Language%20Specification.pdf)

And FYI, when I added this selector, I referenced to `/rules/no-static-only-class.js`'s selector.

Reply:
I don't know if it's possible to have different body in TS, I guess it's fine to keep this just in case.

If you find anything want test, you can use

```js
test.typescript();
// OR
test.snapshot({
	valid: [
		{
			code: '',
			parser: parsers.typescript,
		}
	],
});
```

There are examples in other rules.